profiles/prefix/linux/riscv/make.defaults

```diff
@@ -0,0 +1,11 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+ARCH="riscv"
+CHOST="riscv64-pc-linux-gnu"
+# The base profile sets ACCEPT_KEYWORDS=riscv and we don't have that in prefix.
```

Comment:
This comment is false. The `riscv` arch is unstable only in Gentoo as far as I know.